### DISCUSSION TITLE
Getting this up and running on a SSP-H-20-1G

### DISCUSSION THREAD

**Discussion by XeroX1988** [bikemike/IntexSpaManager#6] [Q&A]
Hi All, Mike,

I'm currently undertaking my first NodeMCU / Arduino adventure. Since we bought the Intex Spa I always wanted to integrate it into Home Assistant to control the thing remotely.

Few days into this i'm discovering there are a couple versions of the spa going round. Here's what I got so far:
* 3d printed the Intex Display Connectors that can be found on thingiverse
* * Had to design the screwable end caps myself, will remix the original thingiverse files and put a link here when done
* Loaded this project on a NodeMCU and it seems I got it working
* Configured Home Assistant with MQTT and verified I got the Hot Tub entity
* Ordered some parts to finish/create a Intex Display Cable to prevent destroying the main / display board

Something I did notice so far and i'd like some input from the community:

I checked the voltage coming off the main board to the display of my spa, this is around 2.5 volts. I don't think this is enough to power the NodeMCU and the display all together. (My NodeMCU doesn't even turn on if I connect the 2.5 volts to a 3v3 pin)

I did notice a 12 volt DC wire on the main board, assuming this is used to trigger a relay to turn the pump on as there's only 12 volt on those wires when I run the pump. If I cannot work with the display cable providing power to the NodeMCU I might want/need to create something that charges a cheap powerbank when the pump us running and have the NodeMCU run of that.

In the schematic of Ulrich Mai i've noticed he's noted his spa is giving 5 volt on the display cable. Is your spa also able to provide 5v on the display cable Mike? Are there any other solutions to get the NodeMCU to run off the 2.5 volts my spa is providing?

**Comment by bikemike** [comment]
That's strange. Mine is 5v. I assume they are all supposed to be 5v.

Are you sure you are measuring the correct two pins? I think the pin labels are R and G.

**Reply by XeroX1988** [reply]
Just checked yours and Ulrich's schematic again. Looks like Ulrich is pulling from the R and G pins, while your schematic looks like it's pulling from B and R pin.

I'll measure the voltage I get from R and G some time this weekend.

**Reply by bikemike** [reply]
Uh oh.. You are right. My schematic is incorrect. I made the schematic after I had already put everything back together and I guess I was looking at the color of the wires and assuming that black was GND.  I'm pretty sure ground is the green one that has the label 'G'. So sorry.

**Reply by XeroX1988** [reply]
Just confirmed, getting 5 volt when using the green wire as ground. Sounds like an assumption I could have made! No worries, no magic smoke got out while figuring this out ;)
